.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  JVM-Run-Gradle-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+
+      - uses: actions/checkout@v3
+
+      # Given we are doing a release, lets make sure we have a safe gradle install
+      - uses: gradle/wrapper-validation-action@v1
+
+      - uses: gradle/gradle-build-action@v2
+
+      - name: Verify all checks pass
+        run: ./gradlew check
+
+      - name: Publish plugin
+        run: ./gradlew publishPlugins
+        env:
+          GRADLE_PUBLISH_KEY: ${{ secrets.GRADLE_PUBLISH_KEY }}
+          GRADLE_PUBLISH_SECRET: ${{ secrets.GRADLE_PUBLISH_SECRET }}
```

Comment:
@joel-costigliola / @scordio we need to add these secrets to this repo. they're found on the [gradle plugin hub](https://plugins.gradle.org/plugin/org.assertj.generator) which I believe @joel-costigliola needs to do since they own this plugin.

Reply:
I have set them with the `gradle.publish.key` and `gradle.publish.secret` values from my gradle account

Reply:
Awesome! Thanks! That was fast